gcf_data_mapper/cli.py

```diff
@@ -1,10 +1,49 @@
+import csv
 import sys
 
 import click
 
 
-@click.command()
+@click.group()
```

Comment:
Not sure we need this as a group - we're probably (?) only going to be generating the data mapper for all of the data at the same time? I can't see why we'd want to do it separately - but I'll raise this as a question with @diversemix